.github/workflows/test.yml

```diff
@@ -50,3 +50,117 @@ jobs:
       if: github.event_name == 'push' && github.ref == 'refs/heads/master'
       env:
         FUZZIT_API_KEY: ${{ secrets.FUZZIT_API_KEY }}
+
+  docker:
+    name: Build and deploy Docker images
+    # Only deploy if previous stages pass.
+    needs: [test, test-linux-alpine, fuzz]
+    runs-on: ubuntu-latest
+    if: >
+      github.event_name == 'push' && 
+        (github.ref == 'refs/heads/master' ||
+         startsWith(github.ref, 'refs/tags/'))
+    env:
+      # Export environment variables for all stages.
+      DOCKER_CLI_EXPERIMENTAL: enabled  # for 'docker buildx'
```

Comment:
nit: please use single spaces for comments. as far as I know, the double-space format is common in python, but not particularly in yaml or Go-related stuff

Reply:
Done

Reply:
> nit: please use single spaces for comments. as far as I know, the double-space format is common in python, but not particularly in yaml or Go-related stuff

actually, yamllint complains about single space comments, doesn't it

Reply:
Yes it does, it issues a warning.
But then, it complains about a bunch of things in this file, mainly wrong indentation of '-' lines, and I didn't want to go there.